### ISSUE TITLE
strange, incomplete polygon rendering at one zoom level and not others

### ISSUE THREAD

**Issue by makella** [CartoDB/carto-vl#1200]
I'm working on some basemap examples and wanted to use a continent layer. What I'm noticing is that at zoom 2, a large chunk of Asia is missing. When I zoom out to 1 or in to 3, the missing portion renders, but then when I'm at zoom 2 again, it disappears.

Not sure if this is isolated to this dataset... haven't noticed it happening with other polygons but just want to point it out in case it is something larger.

thanks!

https://cartodb.github.io/carto-vl/examples/editor/index.html#eyJhIjoiY29udGluZW50IiwiYiI6IiIsImMiOiJjYXJ0b3ZsIiwiZCI6Imh0dHBzOi8ve3VzZXJ9LmNhcnRvLmNvbSIsImUiOiJcblxuXG5cblxuIiwiZiI6eyJsbmciOjg0LjQ5NDc4MjQ5NDM0NDQ2LCJsYXQiOjIuODQyMTcwOTQzMDQwNDAxZS0xNH0sImciOjIsImgiOiJEYXJrTWF0dGVyIiwiaSI6ImRhdGFzZXQifQ==

**Comment by Algunenano** [comment]
This is because, by default, the tiler is oversimplifying polygons before sending them through the MVT functions to match the old behaviour we had with Mapnik (because of a bug). In this case the polygon becomes invalid after the simplification and the clipping process in Postgis ends up with a weird geometry. When we detect one of this geometries in Postgis we currently discard it as otherwise things look pretty awful.

Something similar happens in [this map](https://bl.ocks.org/jsanz/94f8d2e93278b5a376f26e0da533e4ac) around Canada at zoom 2.

This pre/over-simplification is optional and was removed for CartoVL tiles in https://github.com/CartoDB/carto-vl/pull/1148, so this case is working fine in master but not in v1.0.

I also expect this kind of issue to be fixed retroactively in the near future as we are expecting to change the clipping library in Postgis to a more robust one (more tolerant to invalid geometries).

In short, and based on local tests:
- Production Postgis + CartoVL 1.0: Broken [Your case] 
- Production Postgis + CartoVL master: OK
- "Next" Postgis + Carto VL 1.0: OK
- "Next" Postgis + CartoVL master: OK

**Comment by makella** [comment]
thanks for the awesome explanation @Algunenano! glad to hear that it is already taken care of :)
